src/backend/core/src/main/proto/protocol.proto

```diff
@@ -10,6 +10,8 @@ package com.lightbend.statefulserverless.grpc;
 import "google/protobuf/any.proto";
 import "google/protobuf/empty.proto";
 import "google/protobuf/descriptor.proto";
+// This is needed to ensure that the FieldOptions are extended when this file is loaded.
+import "entitykey.proto";
```

Comment:
This is the fix. I had this in my local version of the copied file, which is why it worked for me.

Languages like JavaScript use the extensions to know what fields to include in the output, otherwise they are ignored. So, anything loading `protocol.proto` needs to ensure that `entitykey.proto` is also loaded to ensure it knows about the entity key.

Reply:
:+1: